Rebeka: Happy thoughts project

rebkova opened the pull request:
Link to my deployed project: https://rebeka-happy-thoughts.netlify.app/

I got stuck with making the like button to work. Otherwise everything went rather smooth. I really enjoyed the project and am enjoying working with React a lot. There is still so much to grasp but I'll get there eventually. :)

BamStugendal commented on README.md:
```diff
@@ -1,13 +1,26 @@
 # Happy Thoughts
+A Twitter like app with positive vibes. :sunny:
```
Very good README, very clear and informative! 😄

BamStugendal commented on code/src/urls.js:
```diff
@@ -0,0 +1 @@
+export const THOUGHTS_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts";
```
Nice that you did a separate file for URL 👍

BamStugendal commented on code/src/components/ThoughtInput.js:
```diff
@@ -0,0 +1,49 @@
+            &#10084;&#65039;
+        </span>
+          {" "}Send Happy Thought{" "}
+          <span
+            aria-label="heart"
+            role="img">
+            &#10084;&#65039;
+        </span>
```
Be consistent in how you separate lines. In the ThoughtList component, you separate lines different 😃

BamStugendal commented on code/src/components/thoughts-list.css:
```diff
@@ -0,0 +1,49 @@
+}
+
+.thought-container {
+  border: 2px solid #000000;
+  background-color: #FFFFFF;
```
You can shorten the HEX color code for white to `#fff` and black to `#000`. Less to write 😉

BamStugendal replied:
You can shorten HEX color code for white to `#fff` and black to `#000`. Less to write 😉

BamStugendal commented on code/src/App.js:
```diff
@@ -1,9 +1,59 @@
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="app-container">
+      <ThoughtInput
+        onNewThought={reachNewThought}
+      />
+      {thoughts.map(thought => (
+        <ThoughtsList
+          key={thought._id}
+          thought={thought}
+          happyThought={thought.message}
+          timeStamp={thought.createdAt}
+          nrOfLikes={thought.hearts}
+          onLike={handleLike}
+        />
+      ))}
     </div>
```
Maybe make `<div className="app-container">` to `<section className="app-container">` to make it more semantic? 😄

BamStugendal reviewed (approved):
Good job Rebeka! Very structured code and a good README, liked that you made a bullet list there. I think I will take inspiration from you and do that for my next project 😌

rebkova commented:
Thanks for the explanation regarding s`etInterval()`, Maks. In my simple world I thought I could use it as I did. :) I've removed the line of code now and added `id` to the` input` tag.
